### PR TITLE
Delete refactoring

### PR DESCRIPTION
**Pull request by anantmittal**

### CODE REVIEW

File: scope_shared/scope/schemas/utils/enums.json [hunk 1]
```diff
@@ -209,6 +209,22 @@
 			},
 			"required": ["Flag as safety risk", "Flag for discussion"]
 		},
+		"documentType": {
```
Comment (jayfo) [inline scope_shared/scope/schemas/utils/enums.json:212]: These aren't really "documentType", they're just the set types?

Comment (anantmittal) [reply scope_shared/scope/schemas/utils/enums.json]: Fixed

**Comment by anantmittal** [comment]
Two tests fail on `invoke test.root` with the error message **TypeError: argument of type 'NoneType' is not iterable**:
1. scope_shared/scope/testing/test_database/test_delete_scheduled_activity.py
2. scope_shared/scope/testing/test_database/test_delete_scheduled_assessment.py

**Comment by anantmittal** [comment]
@jayfo Ready for review. Again, all tests pass except the ones I mentioned here https://github.com/uwscope/scope-web/pull/365#issuecomment-1291265414

**Comment by jayfo** [comment]
Marked delete-related tests as `xfail` and `TODO: Activity Refactor` pending later resolution.

**Pull request merged**